CONTRIBUTING.md

```diff
@@ -1,24 +1,32 @@
 # Contributing to Prebid.js
-Contributions are always welcome. To contribute, [fork](https://help.github.com/articles/fork-a-repo/) Prebid.js, commit your changes, and [open a pull request](https://help.github.com/articles/using-pull-requests/).
+Contributions are always welcome. To contribute, [fork](https://help.github.com/articles/fork-a-repo/) Prebid.js, commit your changes, and [open a pull request](https://help.github.com/articles/using-pull-requests/) against the master branch.
 
-## Pull Requests
-Please make sure that pull requests are scoped to one change, and that any added or changed code includes tests with greater than 80% code coverage. See [Testing Prebid.js](#testing-prebidjs) for help on writing tests.
+Details about the Pull Request process can be found [here](./pr_review.md).
```

Comment:
hate to have duplicative information, but I feel like stating 80% code coverage is nice to have here, even if it is in the linked doc.